### PR TITLE
DISPLAY-986: Setup separate image builds for itkdev and os2display

### PR DESCRIPTION
**Pull request by turegjorup**
#### Link to ticket

https://jira.itkdev.dk/browse/DISPLAY-986

#### Description

Setup separate image builds for itkdev and os2display to allow the project to support https://github.com/os2display/os2display-docker-compose going forward while still supporting ITK's setup

#### Screenshot of the result

If your change affects the user interface you should include a screenshot of the result with the pull request.

#### Checklist

- [ ] My code is covered by test cases.
- [ ] My code passes our test (all our tests).
- [ ] My code passes our static analysis suite.
- [ ] My code passes our continuous integration process.

If your code does not pass all the requirements on the checklist you have to add a comment explaining why this change 
should be exempt from the list.

#### Additional comments or questions

If you have any further comments or questions for the reviewer please add them here.

### CODE REVIEW

File: infrastructure/os2display/etc/confd/conf.d/nginx.conf.toml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[template]
+src = "nginx.conf.tmpl"
+dest = "/etc/nginx/nginx.conf"
+mode = "0644"
+keys = [
+    "/nginx-config"
+]
```
Comment (rimi-itk) [inline infrastructure/os2display/etc/confd/conf.d/nginx.conf.toml:7]: Add a newline at end of file for consistency with other `toml` files.

File: infrastructure/itkdev/run.sh
Comment (rimi-itk) [inline infrastructure/itkdev/run.sh:1]: Should `os2display` be replaced with `itkdev` in this file?

File: infrastructure/itkdev/Readme.md
Comment (rimi-itk) [inline infrastructure/itkdev/Readme.md:1]: Should `os2display` be replaced with `itkdev` in this file?

**Review by rimi-itk — CHANGES_REQUESTED** [review]

File: infrastructure/itkdev/run.sh
Comment (turegjorup) [reply infrastructure/itkdev/run.sh:1]: No. `itkdev/os2display-api-service` is the name of the images we have been building for our use.

Comment (rimi-itk) [reply infrastructure/itkdev/run.sh:1]: Aren't we building images with tags like `itkdev/os2display…`?

Comment (turegjorup) [reply infrastructure/itkdev/run.sh:1]: My bad. Looking at the wrong file...

File: infrastructure/itkdev/Dockerfile
[the remark this answers is not in the document]
Comment (turegjorup) [reply infrastructure/itkdev/Dockerfile:1]: No. Looks correct to me

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**